egs/timit/asr/simple_v1/path.sh

```diff
@@ -0,0 +1,3 @@
+export SNOWFALL_ROOT=`pwd`/../../../..
+[ -f $SNOWFALL_ROOT/tools/env.sh ] && . $SNOWFALL_ROOT/tools/env.sh
```

Comment:
What does env.sh contain?

Reply:
Em...It (path.sh) is a unnecessary file. Have deleted.